### ISSUE TITLE
Running a pure Lambda function only in one stage?

### ISSUE THREAD

**Issue by budowski** [aws/chalice#1009]
Let's say I have a pure Lambda function running every 24 hours:
```python
@app.schedule('rate(24 hours)')
def myFunc(event):
    pass
```

How do I make this kind of function run only on one stage? Assuming I deployed all other functions on all stages.

**Comment by wollerman** [comment]
Wrap the whole definition in an if statement that checks the stage.

**Comment by budowski** [comment]
Yeah, that's a good option too. Just thought there might have been a Chalice-based solution

**Comment by kyleknap** [comment]
@budowski there is no built-in chalice solution to get the stage name at runtime/deploy time. I think that is something we would like to add in the future for controlling runtime behavior based on the stage name. Would you mind elaborating on your use-case for wanting to have that lambda function being deployed to only one stage as opposed to all of them?

**Comment by kyleknap** [comment]
Gotcha. Or rather I was referring to chalice actually having a public property for the stage name such an `app.current_stage` property. Right now, the user has to supply the `stage` field in the event payload and this option would allow you to avoid having to do supply it yourself in the event payload. Otherwise, marking as a feature request.

**Comment by budowski** [comment]
Note - if you schedule your lambda (`@app.schedule`), the `event` param doesn't hold the stage name, so there's no easy way of detecting your current stage.
